ios/AirMaps/AIRMap.m

```diff
@@ -104,13 +105,9 @@ - (void)insertReactSubview:(id<RCTComponent>)subview atIndex:(NSInteger)atIndex
     } else if ([subview isKindOfClass:[AIRMapCircle class]]) {
         [self addOverlay:(id<MKOverlay>)subview];
     } else if ([subview isKindOfClass:[AIRMapUrlTile class]]) {
-        ((AIRMapUrlTile *)subview).map = self;
```

Comment:
I don't think you meant to remove these lines. Doing so would remove the ability to add child components inside Views, as well as break urlTile prop switching.